src/components/Home/Header/index.tsx

```diff
@@ -114,94 +124,130 @@ const logosData: Array<{
     widthMd: 88,
     widthLg: 124,
     src: rabbitmqLogo,
-    alt: 'RabbitMQ logo'
+    alt: 'RabbitMQ logo',
+    link: '/doc/user-guide/serving/rabbitmq'
```

Comment:
This page is empty. Maybe not add a link at all? https://mlem.ai/doc/user-guide/serving/rabbitmq

Reply:
@omesser @mike0sv, WDYT about these questions?

Reply:
I would add a link once there is a bit more content (like linking to a working example and some explanations of how it works and how to build an app with it)